build/pxc-entrypoint.sh

```diff
@@ -254,6 +254,12 @@ else
 
 fi
 
+WSREP_CLUSTER_NAME=$(grep wsrep_cluster_name  ${CFG}| cut -d '=' -f 2| tr -d ' ' )
+if [[ !  ${WSREP_CLUSTER_NAME} || ${WSREP_CLUSTER_NAME} == 'noname' ]]; then
+  echo "dns maybe in a error, check it first...."
```

Comment:
@zlcnju I think this message is more understandable:
```suggestion
  echo "Cluster name is invalid, please check DNS"
```

Reply:
yes, it is better.